yarn.lock

```diff
@@ -4552,13 +4561,70 @@
     "@openzeppelin/contracts" "3.0.1"
     ethers "4.0.45"
 
+"@statechannels/wallet-core@0.1.4":
+  version "0.1.4"
+  resolved "https://registry.npmjs.org/@statechannels/wallet-core/-/wallet-core-0.1.4.tgz#b991a1e109ef8f734a6706b5dc9f903322a4b2f7"
+  integrity sha512-l2KyZ+1o1hZdoa1lu2PICSWyUbNLku0ieF00gKlKXFCjOAdfDsbuGOh+xyGfcpQHuL+9lXL17w3v3mfTTBifbg==
+  dependencies:
+    "@statechannels/nitro-protocol" "0.1.5"
+    "@statechannels/wire-format" "0.0.7"
+    ethers "5.0.0-beta.191"
+    lodash "4.17.19"
+
 "@statechannels/wire-format@0.0.7":
   version "0.0.7"
   resolved "https://registry.npmjs.org/@statechannels/wire-format/-/wire-format-0.0.7.tgz#288fa4bdb93df3ea3ff762253106abb9d2a1aaef"
   integrity sha512-D9rFe079FNj5kKil+jj8yAMnrWYAgIZ7UhfJ5JR3GFtHc65jTkMzFjuZUvi+1rJPMRMF0uPWlefDnvd4ulHTYA==
   dependencies:
     ajv "6.11.0"
 
+"@statechannels/xstate-wallet@0.0.8":
+  version "0.0.8"
+  resolved "https://registry.npmjs.org/@statechannels/xstate-wallet/-/xstate-wallet-0.0.8.tgz#fc5e6c82ecba0500c1fafdf6493995fa1c3d5bfb"
+  integrity sha512-xsUwW4MTbhkQgM+KhK2iP5flAwWPgDptTYTPEMzOY2Q5CVGKsmFHqWaj0kdHCZ/TraGMFio+Tl/iX6CelyR9Xw==
+  dependencies:
+    "@material-ui/icons" "4.9.1"
+    "@rimble/connection-banner" "1.2.3"
+    "@rimble/utils" "1.2.3"
+    "@sentry/browser" "5.15.5"
+    "@statechannels/client-api-schema" "0.0.3"
+    "@statechannels/nitro-protocol" "0.1.5"
+    "@statechannels/wallet-core" "0.1.4"
+    "@statechannels/wire-format" "0.0.7"
+    "@xstate/react" "1.0.0-rc.3"
+    async-lock "1.2.2"
+    awesome-typescript-loader "5.2.1"
+    babel-loader "8.0.6"
```

Comment:
See the note here: https://github.com/statechannels/monorepo/blob/276fd42a0f4211ad5d340f9dece05c027cea3357/packages/xstate-wallet/README.md#why-are-webpack-loaders-in-dependencies